### PR TITLE
Create drop-wireless.toml

### PR DESCRIPTION
**Pull request by GiupiDeLuca**

### CODE REVIEW

**Review by eherrerosj — APPROVED** [review]
thx!

File: data/ecosystems/d/drop-wireless.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Ecosystem Level Information
+title = "drop-wireless"
```
Comment (eherrerosj) [inline data/ecosystems/d/drop-wireless.toml:2]: @GiupiDeLuca, can we use "Drop Wireless" for the tittle instead?

**Review by eherrerosj — CHANGES_REQUESTED** [review]
small req

File: data/ecosystems/d/drop-wireless.toml (hunk 1 above)
Comment (GiupiDeLuca) [reply data/ecosystems/d/drop-wireless.toml:2]: Sure, will fix now!

**Pull request merged**